Jenkinsfile

```diff
@@ -994,6 +994,57 @@ pipeline {
       }
     }
 
+    stage('L2: NMT Tarred Dataset Creation') {
+      when {
+        anyOf{
+          branch 'main'
+          changeRequest target: 'main'
+        }
+      }
+      failFast true
+      parallel {
+        stage('L2: NMT Auto Tarred Dataset Creation') {
```

Comment:
Should we want to set `do_training=False` for this test?

Reply:
Done.